apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.sql

```diff
@@ -101,3 +101,13 @@ SET
 WHERE
   iq.assessment_question_id = $assessment_question_id
   AND iq.id = ANY ($instance_question_ids::bigint[]);
+
+-- BLOCK select_ai_grading_job_sequence_ids_for_assessment_question
```

Comment:
nit: can we pull this out into a more generic helper for selecting info from the job sequence table?